ansible/configs/ansible-tower-advanced/sample_vars.yml

```diff
@@ -28,7 +28,7 @@ HostedZoneId: XXXXXXXXXXXXXX            # You will need to change this
 key_name: id_rsa_xxx                    # Keyname must exist in AWS
```

Comment:
@newgoliath why are you touching an ansible-tower-advance file here? All others are ocp4-workload related. Is this a mistake?

Reply:
kk fix for something in the branch as it's not visible in repo